public_html/question1.js

```diff
@@ -0,0 +1,34 @@
+//given n numbers generate an array of length n and put random numbers in the array
+(function () {
+    let arr = [];
+    let answer = prompt("Enter a number");
+    let n = parseInt(answer);
+    for (let i = 0; i < n; i++) {
+        //gives random integers between 1-20 inclusive
+        arr.push(Math.floor(Math.random() * 20));
```

Comment:
Generation of random integer is off by 1. This generates numbers between 0 and 19.